po/es.po

```diff
@@ -2627,14 +2627,10 @@ msgid ""
 "panics in debug mode (`cargo build`) and wrap-around in release mode (`cargo "
 "build --release`)."
 msgstr ""
-"El desbordamiento de enteros se define mediante la bandera de tiempo\n"
-"de compilación [`overflow-checks`](https://doc.rust-lang.org/rustc/codegen-"
-"options/index.html#overflow-checks). Si se habilita, el programa entrará en "
-"_panic_ (un fallo controlado del programa). De lo contrario, se obtendrá "
-"semántica de _wrap-around_. De forma predeterminada, se obtienen _panics_ en "
-"el modo de depuración (`cargo build`) (\"versión de compilación\") y "
-"semántica de _wrap-around_ en el modo de liberación (`cargo build --"
-"release`)."
+"El overflow en Integer es definido vía un flan en tiempo de compilación. Las "
```

Comment:
"flan"? that is a desert. You can use _flag_ in english in italics if you want. The translation is "bandera", and in some countries is used this way too, but not everywhere.